src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -862,5 +862,54 @@ public async Task BreakpointOnHiddenLineOfMethodWithNoNextVisibleLineShouldNotPa
             Task t = await Task.WhenAny(pause_task, Task.Delay(2000));
             Assert.True(t != pause_task, "Debugger unexpectedly paused");
         }
+
+        [Fact]
+        public async Task SimpleStepTest()
+        {
+            await SetBreakpoint("dotnet://debugger-test.dll/debugger-test.cs", 674, 0);
+
+            string expression = "window.setTimeout(function() { invoke_static_method ('[debugger-test] Foo:RunBart'); }, 1);";
+            await EvaluateAndCheck(
+                expression,
+                "dotnet://debugger-test.dll/debugger-test.cs", 674, 12,
+                "Bart");
+            await StepAndCheck(StepKind.Over, "dotnet://debugger-test.dll/debugger-test.cs", 677, 8, "Bart");
+            await StepAndCheck(StepKind.Over, "dotnet://debugger-test.dll/debugger-test.cs", 678, 4, "Bart");
+        }
+
+        [Fact]
+        public async Task StepAndEvaluate()
+        {
+            await SetBreakpoint("dotnet://debugger-test.dll/debugger-test.cs", 682, 0);
+
+            string expression = "window.setTimeout(function() { invoke_static_method ('[debugger-test] Foo:RunBart'); }, 1);";
+            await EvaluateAndCheck(
+                expression,
+                "dotnet://debugger-test.dll/debugger-test.cs", 682, 8,
+                "RunBart");
+            var pause_location = await StepAndCheck(StepKind.Into, "dotnet://debugger-test.dll/debugger-test.cs", 671, 4, "Bart");
+            var id = pause_location["callFrames"][0]["callFrameId"].Value<string>();
+            await EvaluateOnCallFrameAndCheck(id, ("this.Bar", TString("Same of something")));
+            pause_location = await StepAndCheck(StepKind.Into, "dotnet://debugger-test.dll/debugger-test.cs", 673, 8, "Bart");
+            id = pause_location["callFrames"][0]["callFrameId"].Value<string>();
+            await EvaluateOnCallFrameAndCheck(id, ("this.Bar", TString("Same of something")));
+        }
+
+        [Fact]
+        public async Task StepOverWithMoreThanOneCommandInSameLine()
```

Comment:
Do we need a test for a stepping over the last line of a method like:
```csharp
bool foo()
{
  ...
  return a() && b == 123;
}
```

Reply:
added to the test.